test/src/test/java/org/corfudb/integration/CorfuReplicationReconfigurationIT.java

```diff
@@ -215,6 +224,108 @@ public void testActiveClusterReset() throws Exception {
+                                        .build())
+                        .build()
+        );
+
+        Table<StringKey, IntValue, Metadata>  mapStandby = corfuStoreStandby.openTable(
+                NAMESPACE,
+                testStreamName,
+                StringKey.class,
+                IntValue.class,
+                Metadata.class,
+                TableOptions.builder().schemaOptions(
+                                CorfuOptions.SchemaOptions.newBuilder()
+                                        .setIsFederated(true)
+                                        .addStreamTag(ObjectsView.LOG_REPLICATOR_STREAM_INFO.getTagName())
+                                        .build())
+                        .build()
+        );
+
+        // Write batchSize num of entries to active map
+        for (int i = 0; i < batchSize; i++) {
+            try (TxnContext txn = corfuStoreActive.txn(NAMESPACE)) {
+                txn.putRecord(mapActive, StringKey.newBuilder().setKey(String.valueOf(i)).build(),
+                        IntValue.newBuilder().setValue(i).build(), null);
+                txn.commit();
+            }
+        }
+        assertThat(mapActive.count()).isEqualTo(batchSize);
+        assertThat(mapStandby.count()).isZero();
+
+        startActiveLogReplicator();
+        startStandbyLogReplicator();
+        log.info("Replication servers started, and replication is in progress...");
+
+        while (mapStandby.count() != mapActive.count()) {
+            // Wait until data is fully replicated
+        }
+
+        Table<LogReplicationMetadata.ReplicationEventKey, LogReplicationMetadata.ReplicationEvent, Message> eventTable =
+                corfuStoreActive.openTable(LogReplicationMetadataManager.NAMESPACE,
+                        REPLICATION_EVENT_TABLE_NAME,
+                        LogReplicationMetadata.ReplicationEventKey.class,
+                        LogReplicationMetadata.ReplicationEvent.class,
+                        null,
+                        TableOptions.fromProtoSchema(LogReplicationMetadata.ReplicationEvent.class));
+
+        // Check that the event table is empty before performing a forced full sync
+        assertThat(eventTable.count()).isZero();
+
+        Table<ClusterUuidMsg, ClusterUuidMsg, ClusterUuidMsg> configTable = corfuStoreActive.openTable(
+                DefaultClusterManager.CONFIG_NAMESPACE, DefaultClusterManager.CONFIG_TABLE_NAME,
+                ClusterUuidMsg.class, ClusterUuidMsg.class, ClusterUuidMsg.class,
+                TableOptions.fromProtoSchema(ClusterUuidMsg.class)
+        );
+
+        // Perform a forced full sync
+        try (TxnContext txn = corfuStoreActive.txn(DefaultClusterManager.CONFIG_NAMESPACE)) {
+            txn.putRecord(configTable, DefaultClusterManager.OP_ENFORCE_SNAPSHOT_FULL_SYNC,
+                    DefaultClusterManager.OP_ENFORCE_SNAPSHOT_FULL_SYNC, DefaultClusterManager.OP_ENFORCE_SNAPSHOT_FULL_SYNC);
+            txn.commit();
+        }
```

Comment:
Along with using this way to trigger a force snapshot sync, can you also manually enter a few more records to the event Table ? 
Also, can you use the stream listener on the replication status on the sink. Once the listener sees is_data_consistent flag set to true, you can immediately check the event table and ensure that the count is 0